pages/osx/GetFileInfo.md

```diff
@@ -0,0 +1,19 @@
+# GetFileInfo
+
+> Get information about a file in an HFS+ directory.
+
+- See date and time of file creation:
+
+`GetFileInfo -d {{filename}}`
```

Comment:
Perhaps this will read better? Then it's more consistent with existing pages :-)
```suggestion
`GetFileInfo -d {{path/to/filename}}`
```

Also, is there a long form of the `-d` argument here? In tldr pages we prefer to use the long forms of arguments to aid clarity. The same goes for the other examples further down.

Reply:
@sbrl, I don't think there is a long-form for any of them. The man page of GetFileInfo does not list any (and I tried to expand them out myself too). Do let me know if you have any suggestions on how to provide more clarity.

Reply:
No problem - just checking, since some commands do have long forms and others don't :-)